src/Toolkit/Str.php

```diff
@@ -748,8 +748,12 @@ public static function rtrim(string $string, string $trim = ' '): string
      *                         string is too long. Ellipsis is the default.
      * @return string The shortened string
      */
-    public static function short(string $string = null, int $length = 0, string $appendix = '…'): ?string
+    public static function short(string $string = null, int $length = 0, string $appendix = '…'): string
```

Comment:
For consistency:

```suggestion
    public static function short(?string $string = null, int $length = 0, string $appendix = '…'): string
```

The same applies to other methods in this class that accept a string that can be null.

Reply:
I think we should do this in a separate PR for all the places where we allow null. Right now it's totally mixed in multiple classes.